pkg/machines/scripts/install_machine.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
```

Comment:
`install_machine.sh` is very similar to `create_machine.sh`.  I guess only one is used, right?  It's best to delete the other one.

Reply:
It is needed, because of issues with virt-install (previous discussions) 2 Scripts = normal + post-install flow.